pages/markets/[marketid].tsx

```diff
@@ -136,17 +137,16 @@ const Market: NextPage<{
       return <NotFoundPage backText="Back To Markets" backLink="/" />;
     }
 
-    //required to fix title element warning
-    const question = indexedMarket.question;
+    useEffect(() => {
+      if (disputes && marketSdkv2.status === "Disputed") {
+        const lastDispute = disputes?.[disputes.length - 1].toHuman();
```

Comment:
It's best to avoid using `toHuman` if possible as it reduces type safety. Take a look at `getDisputedCategoricalOutcome` to see how you can use stuff like `isCategorical`, `asCategorical`, etc

Reply:
Thanks, I took a look and did something similar and ended up using `toJSON` as this gave me access to the whole disputes object. Does that work or will it cause issues similar as `toHuman`.

Reply:
Yeah `toJSON` can do weird stuff too, from what I remember `toJSON` often encodes strings in a strange way and `toHuman` comma separates numbers